eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -2287,7 +2287,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
 
   case class MyState(state: FSM.State[fr.acinq.eclair.channel.State, Data]) {
 
-    def storing(): FSM.State[fr.acinq.eclair.channel.State, Data] = {
+    def storing(unused: Unit = ()): FSM.State[fr.acinq.eclair.channel.State, Data] = {
```

Comment:
This one is weird, any idea why that is needed?

Reply:
```
Error:(1992, 34) can't supply unit value with infix notation because nullary method storing: ()akka.actor.FSM.State[fr.acinq.eclair.channel.State,fr.acinq.eclair.channel.Data] takes no arguments; use dotted invocation instead: Channel.this.state2mystate(Channel.this.goto(CLOSING).using(nextData)).storing()
    goto(CLOSING) using nextData storing() calling (doPublish(closingTx))
```

Reply:
Got it, so it's to maintain the way we call that with a space, sounds ok